### DISCUSSION TITLE
comments

### DISCUSSION THREAD

**Discussion by giscus** [ximuliunian/blog#4] [General]
# 留言板

曦暮流年 - 一个多姿多彩的个人站点，涵盖技术分享、学习心得、个人博客等内容。在这里，您可以探索编程技巧、获取学习灵感、阅读生活随笔，以及参与互动交流，共同成长。

**Comment by Kesazake** [comment]
为了写下这条留言我专门注册了GitHub的账号。我只是想说，这个个人网站真的非常棒。

**Reply by ximuliunian** [reply]
嘿嘿，谢谢~

**Reply by Kesazake** [reply]
运行这么久，为什么留言板这么空阿。

**Reply by ximuliunian** [reply]
中间进行过好几次迭代，那时候还没有评论这个功能，还有可能就是没什么人来吧＞︿＜

**Reply by Kesazake** [reply]
好可惜。虽然我不懂程序和计算机，但是在我看来，这个网站的设计和审美和画风都真的超棒超有趣。

**Reply by ximuliunian** [reply]
这个网站是使用hexo和Butterfly主题在DIY一下搞出来的，简单来说，就是套模板弄出来的，我喜欢这样的，所以这样的博客就呈现在你们眼前了ヽ(✿▽ﾟ)ノ

**Reply by Kesazake** [reply]
25分钟前我是没有想到您会马上回复的，都这个点了，早点休息吧。

**Reply by ximuliunian** [reply]
嘿嘿，我正好在写新项目的网页，刚写完发布出去，就看到有一个人给我留言，马不停蹄就来啦，确实，都这个点啦，你也早点休息(￣o￣) . z Z

**Reply by Kesazake** [reply]
晚安，祝好梦。

**Reply by ximuliunian** [reply]
晚安(￣o￣) . z Z

**Comment by MJYSG** [comment]
汉化来的，网站真牛逼，建议B站发个视频，肯定有人感兴趣

**Reply by ximuliunian** [reply]
嗯嗯，会考虑的

**Reply by Kesazake** [reply]
同意的，你看你这还有游戏资源，白嫖党狂喜。

**Reply by ximuliunian** [reply]
最近在忙另一件事，等有时间了在搞吧

**Comment by Kesazake** [comment]
有意思的小东西好多阿，不过很多时候我加载一些东西很慢，比如壁纸，还有下载里的病毒那个我打开什么都加载不出来，这个是我的问题嘛？

**Reply by ximuliunian** [reply]
电脑病毒那一栏由于我太懒了，就一直没有上传再加上基本上没什么时间，所以就一直空着了┑(￣Д ￣)┍

**Reply by Kesazake** [reply]
原来如此。

**Reply by ximuliunian** [reply]
至于加载慢，不要怀疑自己，你没有问题，我这个破站加载慢也不是一天两天了(￣m￣）

**Reply by Kesazake** [reply]
没关系，我还是会常来看看。

**Reply by ximuliunian** [reply]
欢迎常来ヾ(^▽^*)))

**Comment by Kesazake** [comment]
想知道站长是做什么的qwq

**Reply by yanxin-ing** [reply]
站长做什么的我不知道，但你这名字好有意思！我没猜错的话是
很高兴认识你
Nice to meet you吧！

**Reply by ximuliunian** [reply]
我啊，现在还在努力吸收知识呢 \(￣︶￣*\))

**Reply by Kesazake** [reply]
搭建这样一个网站会需要很多费用嘛。

**Reply by ximuliunian** [reply]
事实上，除了域名之外，所有东西没花一分钱，之前就连域名都没花钱，但是那个域名被回收了

**Reply by Kesazake** [reply]
这样阿。

**Reply by ximuliunian** [reply]
为什么网站会访问慢，可能跟没花钱的原因有很大关系

**Reply by Kesazake** [reply]
可以理解。毕竟是个非盈利的网站。

**Reply by ximuliunian** [reply]
本来当初建站的目的就是当一个可以随时随地在任何设备上只要有网就可以查看的记事本，后来发现了更多的玩法，从此便一发不可收拾     o(￣▽￣)ｄ

**Reply by Kesazake** [reply]
《记事本》

**Reply by ximuliunian** [reply]
嗯……笔记本

**Reply by Kesazake** [reply]
是指那个上锁的稳定嘛。

**Reply by Kesazake** [reply]
文档。

**Reply by ximuliunian** [reply]
应该吧，没有搞过，在逛GitHub的时候看到，感觉很好玩，就搞了一个     o(￣▽￣)ｄ

**Reply by Kesazake** [reply]
行动派耶qwq

**Reply by ximuliunian** [reply]
嘿嘿，还好啦     ✪ ω ✪

**Reply by Kesazake** [reply]
也许你可以在这个“笔记本”上多记一点各种各样的东西，然后就会有各种各样的人来。

**Reply by ximuliunian** [reply]
事实上我也是这么做的，像是什么白嫖office啊或者刷微软的积分啊什么什么的，但是主要就是不知道更新什么，我认为这个东西值得记录到网站内，那我就记录了，至于各种各样的人的话……都可以的啦

**Reply by Kesazake** [reply]
比如说更个怎么压缩文件qwq。我不知道为什么我使用压缩文件压缩出来的压缩包只有“包”的效果没有“压缩” 的效果。

**Reply by ximuliunian** [reply]
首先先回答你的问题：一般这种情况可能是你选择的压缩等级不对
![image](https://github.com/ximuliunian/ximuliunian.github.io/assets/98019029/0cfefa9f-0fad-4554-9f47-8b28b7430d48)
我这个使用的是7-zip，在下载里面有的
其次我感觉压缩文件这个在我的认知里好像是只要会用电脑就知道的

**Reply by Kesazake** [reply]
你这样显得我很笨qnq。我总感觉压缩出来减少的体积好少，比如220MB压缩完还是有150MB.

**Reply by ximuliunian** [reply]
哈哈，没有没有，你这不少了，压缩有有损压缩和无损压缩，体积小不一定是好事，体积大能很大程度上保证你的数据，比如一张1G的图片有损压缩成1MB那么当你解压的时候原本高清的图片变成马赛克了，但是还是一张1G的图片通过无损压缩成600MB，那么解压之后还是高清的图片

**Reply by Kesazake** [reply]
分卷压缩会对减少体积有帮助吗？

**Reply by ximuliunian** [reply]
我感觉没有，该是多少还是多少，就好比原本五个香蕉压缩之后变成了三个香蕉，分卷压缩就是把这三个香蕉分开放，但是还是改变不了一共是三个香蕉的事实，如果最后成了两个过着一个香蕉了，那么你的数据就丢失一部分了

**Reply by Kesazake** [reply]
那为什么大多数大资源在压缩上传的时候一般都分卷压缩呀？

**Reply by ximuliunian** [reply]
不晓得可能上传快吧，或者是有上传显示，就比如说某度网盘，他的上传限制是4G，一旦上传的文件超过4G他就会禁止这个文件上传，这个时候有一个非常大的文件想要上传，这个时候分卷压缩的好处就来啦，它可以吧大文件分成很多的(<=4)小文件来进行上传

**Reply by Kesazake** [reply]
原来如此。

**Reply by ximuliunian** [reply]
o(￣▽￣)ｄ

**Comment by Kesazake** [comment]
站长，你”网址导航“，”GitHub加速“和”更新日志“怎么没了。

**Reply by ximuliunian** [reply]
没啊，还在呢
![image](https://github.com/ximuliunian/ximuliunian.github.io/assets/98019029/d2bb032b-2f3a-4ef2-852b-046970734319)
可能是没刷新出来吧

**Comment by Xiancang** [comment]
b站视频汉化来的，结果发现大佬做的网址太精妙了！特意注册了GitHub只为赞美，伟大无需多言！~\(≧▽≦)/~

**Reply by ximuliunian** [reply]
哈哈，多谢多谢~ o((>ω< ))o
我也很荣幸得到你的认可 (^^ゞ
